showcases/src/Showcases.ts

```diff
@@ -39,10 +38,12 @@ export class Showcases extends HTMLElement {
       ${acc}
       ${renderComponent({
         componentClass,
-        [showcaseAttr]: showcase,
         hasLongText,
         captionWidth,
         hasCheckeredBackground,
+        ...(!!showcaseClasses
+          ? { showcaseClass: showcase, showcaseStyle: undefined }
+          : { showcaseClass: undefined, showcaseStyle: showcase }),
```

Comment:
`[showcaseAttr]: showcase,` worked pretty well for me
I checked in VSCode and it seems to be valid TS syntax